site/content/kapp/docs/develop/config.md

```diff
@@ -150,6 +150,30 @@ waitRules:
   resourceMatchers:
   - apiVersionKindMatcher: {apiVersion: corp.com/v1, kind: Application}
 ```
+#### waitRules for Custom Resources
+
+Available in v0.48.0+.
+
+Provides a way to add `waitRules` for Custom Resources that don't have `conditions` field in their `status`. This allows users to configure arbitrary rules. For example [this](https://github.com/vmware-tanzu/carvel-kapp/blob/develop/test/e2e/custom_wait_rules_test.go#L79-L88) CR which has `currentState` field inside `status` we can define `waitRule` as shown below, `is_done(resource)` method signature and return type should always remain same while the method implementation can be changes as per the usecase. 
```

Comment:
Should we use something like `As of v0.48.0 ytt wait rules can be used to wait for a resource using any specific field...`? And then we show use the example?

Reply:
I think this is how we mention in the docs at other places as well. `Available in v0.48.0+.`